lib/fpm/package/deb.rb

```diff
@@ -50,6 +50,8 @@ class FPM::Package::Deb < FPM::Package
     value
   end
 
+  option "--dist", "DIST-TAG", "Set the deb distribution."
```

Comment:
Can you set a default value: `:default => "unstable"`

My hope is that this improves documentation (the `--help` shows you the default), and further, it would let you remove one line of code:

(This line below can be removed if you set a default value)
`       distribution = self.attributes[:deb_dist] ? "#{self.attributes[:deb_dist]}" : "unstable"; ` further down in this file.